src/common/intl.cpp

```diff
@@ -513,7 +513,7 @@ bool wxLocale::Init(int lang, int flags)
 /*static*/
 int wxLocale::GetSystemLanguage()
 {
-    return wxUILocale::GetSystemLanguage();
+    return wxUILocale::GetSystemLocale();
```

Comment:
I think this merits a comment saying that we intentionally use the "wrong" function here for compatibility, as otherwise the temptation to "fix" this by changing it to `GetSystemLanguage()` risks being too strong to resist.

Reply:
> I think this merits a comment saying that we intentionally use the "wrong" function here for compatibility, as otherwise the temptation to "fix" this by changing it to `GetSystemLanguage()` risks being too strong to resist.

Yes, you are right. I will add a comment.